chainercv/links/model/faster_rcnn/faster_rcnn_vgg.py

```diff
@@ -75,7 +75,12 @@ class FasterRCNNVGG16(FasterRCNN):
             'url': 'https://github.com/yuyu2172/share-weights/releases/'
             'download/0.0.4/'
             'faster_rcnn_vgg16_voc07_trained_2017_08_06.npz'
-        }
+        },
+        'voc0712': {
+            'n_fg_class': 20,
+            'url': 'https://github.com/yuyu2172/share-weights/releases/'
+            'download/0.0.4/faster_rcnn_vgg16_voc0712_2017_07_21.npz'
```

Comment:
This model is not marked as `_trained_` although VOC07 model is marked. Was this model converted from Caffe or other frameworks?